### ISSUE TITLE
Back buttons removed from focus order within customizer

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-aldine#581]
On book customization pages, the back buttons have a tabindex value of -1 and so are removed from the focus order. These buttons receive focus when the menu is first opened but the user cannot return to these buttons once they interact with the content of the sub menu. 

<img width="607" height="135" alt="Image" src="https://github.com/user-attachments/assets/244dc60f-c14d-41c5-a75a-275639ba5a15" />
